hub/src/server/drivers/diskDriver.js

```diff
@@ -1,5 +1,5 @@
 /* @flow */
-import fs from 'fs'
+import fs from 'fs-extra'
```

Comment:
The changes in this file are part of a different PR.  Please remove this commit from this PR.

Reply:
There are a few minor changes in in this PR that require changes in the previous pending PR https://github.com/blockstack/gaia/pull/158 - easy to fix but maybe lets get that PR resolved first then I can update this one to accommodate?